src/main/java/core/basesyntax/service/impl/FileReportCreatorImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReportCreator;
+import java.io.File;
+
+public class FileReportCreatorImpl implements FileReportCreator {
```

Comment:
I think this class should not create some FIle, but create String with report info